transmogrifier/sources/transformer.py

```diff
@@ -131,25 +317,25 @@ def get_required_fields(self, xml: Tag) -> dict:
         }
 
     @final
-    def transform(self, xml: Tag) -> Optional[TimdexRecord]:
+    def transform(self, source_record: Tag) -> Optional[TimdexRecord]:
         """
         Transform an OAI-PMH XML record into a TIMDEX record.
```

Comment:
Feels like an opportunity to remove "OAI-PMH" where it pops up and be generic that it's just "XML".

Reply:
Missed this one!